stream-chat-android-sample/src/main/kotlin/io/getstream/chat/sample/application/ExtraDependencies.kt

```diff
@@ -0,0 +1,8 @@
+package io.getstream.chat.sample.application
+
+import android.app.Application
+
+interface ExtraDependencies {
+
+    fun configFlipper(application: Application)
```

Comment:
We could have just a single `config` function in this class, so that this interface doesn't reference `Flipper` directly